src/components/ItemDetail/use/useLogs.ts

```diff
@@ -17,7 +17,10 @@ const useLogs = (props: {
     props.logs.map(v => {
       const userName = v.user.name
       const ownerWord =
-        v.ownerId === traP_ID || v.type > 1 ? '' : `${v.owner.name}さんの`
+        v.ownerId === traP_ID ||
+        (v.type !== LogType.add && v.type !== LogType.remove)

```

Comment:
```suggestion
        (v.type === LogType.add || v.type === LogType.remove)
```
こうじゃない？(addとremoveって2と3だよね？)

Reply:
おばかちんだった